### PR TITLE
Add support for canvas layers

### PR DESCRIPTION
**Pull request by aratcliffe**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (tmcw) [inline .gitignore]: please don't add this: a node_modules exclude should be part of your global gitignore.

File: index.js [hunk 2]
```diff
@@ -91,13 +92,26 @@ module.exports = function leafletImage(map, callback) {
                 .add(origin);
 
             if (tilePoint.y >= 0) {
-                var url = addCacheString(layer.getTileUrl(tilePoint));
-                tileQueue.defer(loadTile, url, tilePos, tileSize);
+	            if (isCanvasLayer) {
```
Comment (tmcw) [inline index.js]: please match surrounding code style - this indentation is different.

File: index.js [hunk 3]
```diff
@@ -44,7 +44,8 @@ module.exports = function leafletImage(map, callback) {
     }
 
     function handleTileLayer(layer, callback) {
-        var canvas = document.createElement('canvas');
+	    var isCanvasLayer = (layer instanceof L.TileLayer.Canvas),
```
Comment (tmcw) [inline index.js]: this value is only used once - please just write `if (layer instanceof L.TileLayer.Canvas)` directly.

File: .gitignore (hunk 1 above)
Comment (aratcliffe) [inline .gitignore]: ok cool. i'm kinda new to node so will set this up globally

File: index.js (hunk 3 above)
Comment (aratcliffe) [inline index.js]: it's used in the tiles.forEach loop on line 95.

**Comment by aratcliffe** [comment]
Have re-formatted the code. Thanks for bringing that to my attention.

**Pull request merged**